dali/test/python/test_operator_affine_transforms.py

```diff
@@ -512,3 +516,151 @@ def verify_deprecation(callback):
 def test_transform_translation_deprecation():
     verify_deprecation(lambda : fn.transform_translation(offset=(0, 0)))
     verify_deprecation(lambda : ops.TransformTranslation(offset=(0, 0))())
+
+
+def test_sequences():
+    np_rng = np.random.default_rng(12345)
+    rng = random.Random(42)
+    num_iters = 4
+    max_num_frames = 50
+    max_batch_size = 12
+
+    class TransformsParamsProvider(ParamsProvider):
+        def unfold_output_layout(self, layout):
+            unfolded = super().unfold_output_layout(layout)
+            if unfolded == "**":
```

Comment:
Why do we get "**"?

Reply:
Transform ops propagate the information about sequence layout to reduce the need for fn.per_frame in the intermediate steps when composing multiple transforms. So if you specify args per-frame, the output matrices have F** layout. The base unfold_output_layout unfolds the outputs (of the layout F**, ) and drops the prefix of the layout (F** -> **). Usually it is OK, because it means something like FHWC -> HWC, but "**" is meaningless and baseline transform op run with no per frame input produces outputs with no layout.